CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 ## DFX
 
+### feat: updates the dfx new starter project for env vars
+
+- Updates the starter project for env vars to use the new `dfx build` & `dfx deploy` environment variables
+- Changes the format of the canister id env vars to be `CANISTER_ID_<canister_name_uppercase>`, for the frontend declaraction file to be consistent with the dfx environment variables. `CANISTER_ID` as both a prefix and suffix are supported for backwards compatibility.
+
+> Migration: If your frontend project is using the old environment variables, you will need to update your `webpack.config.js` file to use the new `CANISTER_ID_prefix` variables, or to use the new webpack config provided in the starter project.
```

Comment:
Please add this part to a new file in https://github.com/dfinity/sdk/tree/master/docs/migration